lib/provider/default_volume_provider_test.go

```diff
@@ -111,8 +111,8 @@ func TestOrderSnapshot(t *testing.T) {
 func TestCreateSnapshot(t *testing.T) {
 	ccf := &DefaultVolumeProvider{sess: nil}
 
-	volume, _ := ccf.CreateSnapshot(&Volume{}, nil)
-	assert.Nil(t, volume)
+	snapshot, _ := ccf.CreateSnapshot("vol-id", SnapshotParameters{})
+	assert.Nil(t, snapshot)
 }
```

Comment:
Needs verification of response

Reply:
Checked implementation of other UTs. Looks like since it is only interface library, we have verified method signature here. Detailed response verification is done in volume-vpc code base